elasticdl/docker/Dockerfile.dev

```diff
@@ -49,11 +49,11 @@ RUN /install-protobuf.bash && rm /install-protobuf.bash
 
 # Install elasticdl.org/elasticdl Go package
 ENV ELASTICDLPATH $GOPATH/src/elasticdl.org/elasticdl
+ENV TF_PATH /tmp/tensorflow
+RUN cd /tmp && git clone --depth=1 https://github.com/tensorflow/tensorflow.git
```

Comment:
Adding this line would significantly slow down the Docker build speed. @QiJune please follow up https://github.com/sql-machine-learning/elasticdl/issues/1731 with @typhoonzero to use Docker build cache as much as possible.

Reply:
I would prefer to fix #1731 before merging this PR; otherwise, the merge of this PR would significantly slow down the CI until we fix #1731.

Reply:
@wangkuiyi  I compare two CI time of two PRs, [PR 1726](https://travis-ci.org/sql-machine-learning/elasticdl/builds/649426307?utm_source=github_status&utm_medium=notification) and [current PR](https://travis-ci.org/sql-machine-learning/elasticdl/builds/649913101?utm_source=github_status&utm_medium=notification)

- PR 1726:  6min 15s + 17min 34s = 23min 49s
- current PR: 6min 6s + 16min 49s = 22min 55s

It seems that downloading tensorflow will not influence the CI speed.

I use`git clone --depth=1`, so the downloading speed will be fast.